src/ForkCMS/Bundle/InstallerBundle/Tests/Controller/InstallerControllerTest.php

```diff
@@ -44,7 +44,8 @@ public function testInstallationProcess()
 
     /**
      * @param \Symfony\Component\DomCrawler\Crawler|null $crawler
-     * @param \Symfony\Bundle\FrameworkBundle\Client $client
+     * @param \Symfony\Bundle\FrameworkBundle\Client|null $client
```

Comment:
Why the `|null` part? (in both crawler and client?) We always expect a Crawler and a Client object. With null, our methods won't work.